scripts/us_census/acs5yr/subject_tables/common/data_loader_test.py

```diff
@@ -12,9 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """ Test for the data loader for the subject tables"""
-import unittest
+
 import os
-from .data_loader import process_subject_tables
+import sys
+import unittest
+
+# Allows the following module imports to work when running as a script
+# relative to data/scripts/
+sys.path.append(
+    os.path.sep.join([
+        '..' for x in filter(lambda x: x == os.path.sep,
+                             os.path.abspath(__file__).split('scripts/')[1])
+    ]))
+
+from data_loader import process_subject_tables
```

Comment:
In cloudbuild, the import path is not resolved completely. The import for module import in line 28 fails with `ModuleNotFoundError` because the `data_loader` module is not found in the path.

We can resolve this by:
1. prefixing `data_loader` module's path with `us_census.acs5yr.subject_tables.common.data_loader`
2. we could also defer the relative path to `data/scripts` change to a later PR and get the geo changes in this PR